src/olympia/shelves/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+from olympia.amo.models import ModelBase
+
+CHOICE_TYPES = (
+    ('category', 'category'),
+    ('collection', 'collection'),
+    ('extension', 'extension'),
+    ('recommended', 'recommended'),
+    ('search', 'search'),
+    ('theme', 'theme'),
+)
+
+
+class Shelf(ModelBase):
+    title = models.CharField(max_length=200, unique=True)
+    shelfType = models.CharField(
+        max_length=200, choices=CHOICE_TYPES, verbose_name='type')
+    criteria = models.CharField(
+        max_length=200,
+        help_text="e.g., search/?recommended=true&sort=random&type=extension")
+    footerText = models.CharField(
+        max_length=200, default="See more", verbose_name="footer text")
```

Comment:
lets not specify the `default` here - it makes it annoying to change (needing database migrations) if we were to update it later from "See more" to a different string.  Just have `blank=True`.  In the serializer we can check if it's not defined and return `'See more'` there instead.

Reply:
Add help_text mentioning the default.